Makefile

```diff
@@ -94,6 +94,7 @@ FV_NUM_BATCHES?=1
 # (with FV_NUM_BATCHES=1) to check that it's not a flake.
 FV_BATCHES_TO_RUN?=$(shell seq $(FV_NUM_BATCHES))
 FV_SLOW_SPEC_THRESH=90
+FV_RACE_DETECTOR_ENABLED?=true
```

Comment:
Can we default this to false for the moment? The fv's are likely to fail right now, aren't they?

Reply:
Realised this is fine for OS and we'll disable it for enterprise